model/registry/vcs.yaml

```diff
@@ -83,3 +94,22 @@ groups:
           The ID of the change (pull request/merge request) if applicable. This
           is usually a unique (within repository) identifier generated by the VCS system.
         examples: ["123"]
+      - id: vcs.change.state
+        type:
+          members:
+            - id: open
+              value: open
+              brief: "[open](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefbranchabranch)"
+              stability: experimental
+            - id: closed
+              value: closed
+              brief: "closed"
+              stability: experimental
+            - id: merged
```

Comment:
In GitHub, there is no merged state actually. It's only open or closed https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests

The way you determine if it was merged is by looking at the `merge_commit_sha` & `merged_at` values.

Reply:
I only say that to say, merged is actually a closed state. https://docs.gitlab.com/ee/api/merge_requests.html#list-merge-requests gitlab on the other hand does have a dedicated merged state. i'd say closed, opened, and merged are well known enums so I would be fine with those three.